src/FilterRow.tsx

```diff
@@ -4,28 +4,28 @@ import classNames from 'classnames';
 import { CalculatedColumn, Filters } from './common/types';
 import { DataGridProps } from './DataGrid';
 
-type SharedDataGridProps<R> = Pick<DataGridProps<R, never>,
+type SharedDataGridProps<R, SR = never> = Pick<DataGridProps<R, never, SR>,
```

Comment:
I agree with Nico's comment. We do not need ` = never` here. Moreover we already have `SR` on line 20